src/components/DownloadContactListButton.tsx

```diff
@@ -14,8 +14,15 @@ export function DownloadContactListButton (props: DownloadButtonProps) {
     const uniqueProperties = getUniqueListBy(arrayOfPropertyObjects, 'id')
     const csvRows = uniqueProperties.map((result: UnprocessedResultsFromCRM) => {
         if (result.owner_details && Array.isArray(result.owner_details)) {
-            const mobile = result.owner_details[0].Mobile
-            const workPhone = result.owner_details[0].Work_Phone
+            let mobile
+            let workPhone
+            if (!result.owner_details[0]) {
```

Comment:
I think this has the same problem as before where it doesn't correctly handle more than 2 owners. I would suggest using Array.prototype.find:

```
const { mobile, phone } = result.owner_details.find((owner) => owner.Mobile && owner.Work_Phone);
```